src/utils/require-template.js

```diff
@@ -19,30 +19,34 @@ function isValidTemplate(template){
   ].every((method) => template[method]);
 }
 
+const getOcTemplate = (path) => {
+  if (require.cache && !!require.cache[path]) {
+    delete require.cache[path];
+  }
+  return require(path);
+};
 
-module.exports = function(template) {
+module.exports = function (template) {
   let ocTemplate;
   const localTemplate = path.join(__dirname, '../../', 'node_modules', template);
   const relativeTemplate = path.resolve('.', 'node_modules', template);
 
   try {
-    if (require.cache && !!require.cache[localTemplate]) {
-      delete require.cache[localTemplate];
-    }
-    ocTemplate = require(localTemplate);
-  } catch(err) {
+    ocTemplate = getOcTemplate(template);
```

Comment:
very good; we are getting there 😉 thanks for your help!

// cc @matteofigus @nickbalestra